src/modules/Thread/components/ParentMessageInfo/index.tsx

```diff
@@ -339,6 +339,11 @@ export default function ParentMessageInfo({
           hideMenu={() => {
             setShowMobileMenu(false);
           }}
+          deleteMenuState={
+            allThreadMessages?.length === 0
+              ? 'ACTIVE'
+              : 'HIDE'
+          }
```

Comment:
Q. what does it mean by this? thread message's length is 0 then Active if not, hide?